exercises/04-integration/after/Divergent.Customers/ProgramService.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.ComponentModel;
+using System.ServiceProcess;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+using Divergent.Customers.Config;
+
+namespace Divergent.Customers
+{
+    [DesignerCategory("Code")]
+    class ProgramService : ServiceBase
+    {
+        IEndpointInstance endpoint;
+        static ILog logger;
```

Comment:
Both these should be prefixed by `private`.

Reply:
I'm used to prefix with private as well, so I'm fine with the suggestion. But isn't it redundant? fields are private by default.

Reply:
I prefer to always be explicit. Different types of members have different access defaults and I'm loathed to have to map back to the default in my mind while I'm reading the code.

Reply:
`class` should be `internal` then 😉

Reply:
Yep, good catch! 😄